cms/envs/common.py

```diff
@@ -836,11 +836,14 @@
     # Self-paced course configuration
     'openedx.core.djangoapps.self_paced',
 
-    # OAuth2 Provider
+    # django-oauth2-provider
     'provider',
     'provider.oauth2',
     'edx_oauth2_provider',
 
+    # django-oauth-toolkit
+    'oauth2_provider',
```

Comment:
Side comment: This is definitely one thing I don't like about Python.  Why don't they just have us put the namespaced name of the app?